k8s/0010.deployment.yml

```
@@ -0,0 +1,20 @@
+    matchLabels:
+      app: dd-cleaner
+  template:
+    metadata:
+      labels:
+        app: dd-cleaner
+    spec:
+      # This deployment doesn't do much, but it satisfies the requirement.
+      containers:
+      - name: do-nothing
+        image: busybox
+        command: ["sh", "-c", "echo 'Placeholder deployment is up'; sleep 3600"]
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)